### PR TITLE
add extra changes needed to get akka cluster support

### PR DESCRIPTION
**Pull request by pjfanning**
see my comments in https://github.com/apache/incubator-pekko/issues/108

### CODE REVIEW

**Comment by pjfanning** [comment]
@mdedetrich would it be possible to review this?

**Comment by mdedetrich** [comment]
> @mdedetrich would it be possible to review this?

You made the PR against the wrong repo (this is `mdedetrich/incubator-pekko`)

**Comment by mdedetrich** [comment]
> @mdedetrich would it be possible to review this?

You made the PR against the wrong repo (this is `mdedetrich/incubator-pekko`)

**Comment by mdedetrich** [comment]
> @mdedetrich would it be possible to review this?

Will do

**Comment by pjfanning** [comment]
> > @mdedetrich would it be possible to review this?
> 
> You made the PR against the wrong repo (this is `mdedetrich/incubator-pekko`)

This will add the changes to your PR.

File: cluster/src/main/scala/org/apache/pekko/cluster/SeedNodeProcess.scala [hunk 1]
```diff
@@ -65,8 +80,14 @@ private[cluster] abstract class SeedNodeProcess(joinConfigCompatChecker: JoinCon
     val configToValidate =
       JoinConfigCompatChecker.filterWithKeys(requiredNonSensitiveKeys, context.system.settings.config)
 
+    val adjustedConfig = if (needsAkkaConfig)
+      ConfigUtil.addAkkaConfig(configToValidate, akkaVersion)
+    else configToValidate
+
+    logInfo("init-join config = " + adjustedConfig)
```
Comment (mdedetrich) [inline cluster/src/main/scala/org/apache/pekko/cluster/SeedNodeProcess.scala:87]: @pjfanning `init-join` seems to be devoid of relevant/important info here, should this be updated?

Comment (pjfanning) [reply cluster/src/main/scala/org/apache/pekko/cluster/SeedNodeProcess.scala]: removed the log statement - was useful when debugging but not useful now

**Pull request merged**